### PR TITLE
Speech to text

### PR DESCRIPTION
**Pull request by claytonhenrylewis**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +17,17 @@
...
+  "dependencies": {
     "electron": "^4.0.1",
-    "electron-tooltip": "^1.1.5"
+    "electron-tooltip": "^1.1.5",
+    "@google-cloud/speech": "^2.2.0",
+    "@google-cloud/storage": "^2.4.1",
+    "jquery": "^3.3.1",
+    "read-blob": "^1.1.0",
+    "recorder-js": "^1.0.7",
+    "web-audio-beat-detector": "^7.0.26"
```
Comment (chang382) [inline package.json]: We can remove this dependency I don't think we use it anymore?

Comment (claytonhenrylewis) [reply package.json]: Aye

**Comment by JustRia** [comment]
LGTM!

**Review by JustRia — APPROVED** [review]
private key stuff is fine because now we are a private club! wow so exclusive

**Pull request merged**